src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemDecisionPageViewModel.kt

```diff
@@ -9,7 +9,12 @@ class SoftwareSystemDecisionPageViewModel(
 ) : SoftwareSystemPageViewModel(generatorContext, softwareSystem, Tab.DECISIONS) {
     override val url = url(softwareSystem, decision)
 
-    val content = markdownToHtml(this, decision.content, generatorContext.svgFactory)
+    val content = markdownToHtml(this, fixADRLinks(decision.content, softwareSystem), generatorContext.svgFactory)
+
+    private fun fixADRLinks(content: String, softwareSystem: SoftwareSystem) =
```

Comment:
I wouldn't call this 'fix' since it is not technically broken. What about an extension method `withTransformedAdrLinks` or similar?